### ISSUE TITLE
old link for invoke-cmd  

### ISSUE THREAD

**Issue by BartekAtEQTek** [dsccommunity/SqlServerDsc#1737]
Hi,

I spotted old link in SqlServerDesc wiki, precisely here: https://github.com/dsccommunity/SqlServerDsc/wiki/SqlScript, invoke-command link directs to 404 page.

best regards,
Bartek

**Comment by johlju** [comment]
Yes, seems this is the new link https://docs.microsoft.com/en-us/powershell/module/sqlserver/invoke-sqlcmd
